examples/aws-xray/src/client.rs

```diff
@@ -13,14 +15,12 @@ fn init_tracer() {
             id_generator: Box::new(XrayIdGenerator::default()),
             ..Default::default()
         })
-        .install();
-
-    global::set_text_map_propagator(XrayTraceContextPropagator::new());
+        .install()
 }
 
 #[tokio::main]
 async fn main() -> std::result::Result<(), Box<dyn std::error::Error + Send + Sync>> {
-    init_tracer();
+    let _guard = init_tracer();
```

Comment:
Is it worth destructuring here to make it explicit which of the parts is the guard? It's a similar case for the other examples. Or maybe we shouldn't worry about it too much.

```suggestion
    let (_tracer, _guard) = init_tracer();
```